its/plugin/src/test/java/com/sonarsource/slang/MeasuresTest.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * SonarSource SLang
+ * Copyright (C) 2009-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package com.sonarsource.slang;
+
+import java.util.List;
+import org.junit.Test;
+import org.sonarqube.ws.Issues;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class MeasuresTest extends TestBase {
+
+  private static final String BASE_DIRECTORY = "projects/measures/";
+
+  @Test
+  public void kotlin_measures() {
+    ORCHESTRATOR.executeBuild(getSonarScanner(BASE_DIRECTORY, "kotlin"));
+
+    assertThat(getMeasureAsInt("empty_file.kt", "lines")).isEqualTo(1);
+    assertThat(getMeasureAsInt("file1.kt", "lines")).isEqualTo(14);
+    assertThat(getMeasureAsInt("file2.kt", "lines")).isEqualTo(12);
```

Comment:
"lines" metric is the responsibility of SQ, and I don't think we should test it.

Reply:
Ok. Removed it in 1f44432452b84a70f79748bd21019f97ee7e0e85